command/auth_tune.go

```diff
@@ -306,6 +319,14 @@ func (c *AuthTuneCommand) Run(args []string) int {
 		if fl.Name == flagNameIdentityTokenKey {
 			mountConfigInput.IdentityTokenKey = c.flagIdentityTokenKey
 		}
+
+		if fl.Name == flagNameTrimRequestTrailingSlashes {
+			mountConfigInput.TrimRequestTrailingSlashes = c.flagTrimRequestTrailingSlashes
+		}
+
+		if fl.Name == flagNameTrimRequestTrailingSlashes {
```

Comment:
Duplicated if fl.Name..

Reply:
Yeah made a bunch of mistakes using git diff + patch to bring changes over and having them conflict.